src/test/scala/org/broadinstitute/dsde/firecloud/webservice/TrialApiServiceSpec.scala

```diff
@@ -737,5 +768,7 @@ object TrialApiServiceSpec {
     enrolledUser -> enrolledUserRegInfo,
     terminatedUser -> terminatedUserRegInfo,
     finalizedUser -> finalizedUserRegInfo,
-    registeredUser -> registeredUserRegInfo)
+    registeredUser -> registeredUserRegInfo,
+    invitedUser -> invitedUserRegInfo
+  )
```

Comment:
this entire spec is a bit messy - the mock data setup and mock dao logic are rather scattered. I'm actively trying to not refactor the whole thing to avoid scope creep.